doc(Fees): Add doc for fee advance in invoice (#170)
getlago/lago-doc: api-reference/plans/object.mdx

        </ResponseField>
        <ResponseField name="pay_in_advance" type="boolean">
          This field determines the billing timing for this specific usage-based
          charge. When set to true, the fee is created and invoices at the event

>           reception. If false the fee is created at event reception but its
>           invoicing will follow the group_paid_fees settings.

Reviewer: This is for `invoiceable` attribute.

```suggestion
          reception. If false the fee will be part of the next subscription invoice
```

        </ResponseField>
        <ResponseField name="invoiceable" type="boolean">
          This field specifies whether the charge should be included in a proper
          invoice. If set to `false`, no invoice will be issued for this charge.
          You can only set it to `false` when `pay_in_advance` is `true`.
        </ResponseField>
        <ResponseField name="regroup_paid_fees" type="string">

>           This setting can only be configured if pay_in_advance is false. This

Reviewer: ```suggestion
          This setting can only be configured if pay_in_advance is true and invoiceable is false. This
```
